test/Microsoft.Dnx.TestHost.Tests/OSSkipConditionFacts.cs

```diff
@@ -11,41 +11,49 @@ namespace Microsoft.Dnx.TestHost.Tests
 {
     public class OSSkipConditionFacts
     {
+        private IRuntimeEnvironment RuntimeEnvironment
```

Comment:
I might be wrong but I think there's a test helper that gives you the runtime environemnt